openrr-planner/src/collision/robot_collision_detector.rs

```diff
@@ -166,3 +166,77 @@ fn test_robot_collision_detector() {
         ]);
     assert!(robot_collision_detector.is_self_collision_detected());
 }
+
+/// Lists all potentially-colliding pairs from a robot chain
+///
+/// robot: robot model
+pub fn create_all_collision_pairs<N>(robot: k::Chain<N>) -> Vec<(String, String)>
```

Comment:
Do you have any reason to use k::Chain instead of &k::Chain?

Reply:
No, so I fix it.